geomesa-tools/conf/application.conf

```diff
@@ -78,7 +91,29 @@ geomesa {
         { name = "vis",      transform = "$9::string"                  }
       ]
     }
-    "example-xml" = {
+    "example-csv-per-attr-visibilities" = {
+      type   = "delimited-text"
+      format = "CSV"
+      options {
+        skip-lines = 1
+      }
+      id-field = "toString($fid)",
+      user-data = {
+        geomesa.feature.visibility = "'owl,owl,owl,newt,newt,owl'"
```

Comment:
are the single quotes required and/or problematic? we should probably update the docs examples if you actually need them.